### DISCUSSION TITLE
How to set up the population with hepatic impairment version 11

### DISCUSSION THREAD

**Discussion by gapa84** [Open-Systems-Pharmacology/Forum#1538] [Questions & Problems]
hello. I am currently creating a PPI model using pk-sim. I want to apply the model by creating a population with hepatic impairment. The previously posted question, [ #1341 Hepatic impairment population ], referred to the answer in this article. However, the potar vein value cannot be modified. Where and how can this be corrected? And how do I set other oragns?
Also, how can I apply the settings for child-pugh B & C?
First of all, I will capture and share the parameters I touched to create a liver disorder patient. For reference, the race setting is made by setting it to Japanese.

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/92776578/e140221b-18e3-41af-be97-502adac45da0)

Thank you in advance.
You must be busy, but please reply!

**Comment by Yuri05** [comment]
Please check if the discussion https://github.com/Open-Systems-Pharmacology/Forum/discussions/1341 answers your questions.

If you are brave, you can also try the development version of PK-Sim 12, where the hepatic impairment population is **built in**
(however be aware that the implementation in V12 was not yet validated!)
You can download the portable version of PK-Sim 12 here: https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/pk-sim-osmoses/artifacts/pk-sim-portable-setup.zip?pr=false (no setup required; can be used in parallel to the installed OSP Suite)

**Reply by gapa84** [reply]
I'm really sorry, but when I opened the file, the version 12 you mentioned is not there. Didn't you give me the wrong file?
A more detailed explanation please.

**Reply by StephanSchaller** [reply]
The link to V12 should directly start the download (--> check your download folder).

**Reply by Yuri05** [reply]
@gapa84 OK, I see, the link above points to 11.2...
Download from here: https://ci.appveyor.com/project/open-systems-pharmacology-ci/pk-sim-osmoses/build/artifacts

**Reply by gapa84** [reply]
very appreciated :)
